src-tauri/src/manager.rs

```diff
@@ -53,11 +56,47 @@ impl ManagerState {
         self.watchers_running.insert(name.to_string(), true);
         self.watchers_pid.insert(name.to_string(), pid);
         println!("{:?}", self.watchers_running);
+        self.update_tray_menu();
     }
     fn stopped_watcher(&mut self, name: &str) {
         println!("stopped {name}");
         self.watchers_running.insert(name.to_string(), false);
         self.watchers_pid.remove(name);
+        self.update_tray_menu();
+    }
+    fn update_tray_menu(&mut self) {
+        let open = CustomMenuItem::new("open".to_string(), "Open");
+        let quit = CustomMenuItem::new("quit".to_string(), "Quit");
+        let mut module_menu = SystemTrayMenu::new();
+
+        for (module, running) in self.watchers_running.iter() {
+            let label = format!(
+                "{} ({})",
+                module,
+                if *running { "Running" } else { "Stopped" }
+            );
+            module_menu = module_menu.add_item(CustomMenuItem::new(module.clone(), &label));
+        }
+
+        let module_submenu = SystemTraySubmenu::new("Modules", module_menu);
+        let menu = SystemTrayMenu::new()
+            .add_item(open)
+            .add_native_item(SystemTrayMenuItem::Separator)
+            .add_submenu(module_submenu)
+            .add_native_item(SystemTrayMenuItem::Separator)
+            .add_item(quit);
+
+            let &(ref lock, ref cvar) = &*SHARED_CONDVAR;
```

Comment:
Interesting `&*`, is it needed?

Reply:
Need to deference out of the struct. Could still be rewritten without it. Does it have any performance implication?

Reply:
Probably not, just curious why it was needed.

Reply:
Clippy gave me this: 
<img width="829" alt="image" src="https://github.com/ActivityWatch/aw-tauri/assets/1405370/704b6440-f3d2-4b46-8990-224ad7be05a0">

Reply:
Let me check on that... this bit was generated by copilot, I didn't check on it that much, since it looked identical to  what I had read in docs.rs. But let me try removing these and see.

Reply:
It's safe to remove.
```rust
let (lock,cvar) = &*SHARED_CONDVAR;
```
works just fine!

Reply:
Auto-fixed it with clippy :)